packages/core/src/config/vendure-config.ts

```diff
@@ -106,6 +107,22 @@ export interface ApiOptions {
      * @default false
      */
     shopApiDebug?: boolean;
+    /**
+     * @description
+     * Custom functions to use as additional validation rules when validating the schema for the admin GraphQL API
+     * [ApolloServer playground](https://www.apollographql.com/docs/apollo-server/api/apollo-server/#constructoroptions-apolloserver).
+     *
+     * @default []
+     */
+    adminApiValidationRules?: ((context: ValidationContext) => any)[];
+    /**
+     * @description
+     * Custom functions to use as additional validation rules when validating the schema for the shop GraphQL API
+     * [ApolloServer playground](https://www.apollographql.com/docs/apollo-server/api/apollo-server/#constructoroptions-apolloserver).
```

Comment:
These links seem to be mis-labeled, and is it supposed to link to https://www.apollographql.com/docs/apollo-server/api/apollo-server/#validationrules?

Reply:
Good catch, should be fixed now